homeassistant/components/withings/__init__.py

```diff
@@ -116,12 +47,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         webhook_id = webhook.async_generate_id()
         config_updates["data"] = {
             **entry.data,
-            **{
-                const.CONF_USE_WEBHOOK: hass.data[DOMAIN][const.CONFIG][
-                    const.CONF_USE_WEBHOOK
-                ],
-                CONF_WEBHOOK_ID: webhook_id,
-            },
+            const.CONF_USE_WEBHOOK: False,
```

Comment:
One thing to note, we're now removing the option to use a webhook. Previously the only way to use them is to actually initialize with it set to true in your yaml. Only on the first load of the integration.

In future follow up PRs I am migrating this option to the options part of the config entry, and in another follow up I will add a config flow for it.

Reply:
If I did all these changes in 1 PR, this would be huuuuuuge. I am planning to make the PRs in the nearby future. (probably almost immediatly after this gets merged)

Reply:
FYI. Removal of webhook endures functionality like in bed detection will be broke. Not that it worked well in practice anyway.

Reply:
Spoke to Edenhaus, will first work on the config flow where the option will be moved to the options part. After that I will create a Options flow, after which the YAML will be removed.